src/libcore/alloc.rs

```diff
@@ -1153,17 +1003,16 @@ pub unsafe trait AllocRef {
     /// function; clients are expected either to be able to recover from
     /// `shrink_in_place` failures without aborting, or to fall back
     /// on another reallocation method before resorting to an abort.
+    #[inline]
```

Comment:
Why is this inline when none of the other functions are?

Reply:
Both cases of `#[inline]` are no-ops and just return `Err`.